src/test/java/harness/baseclasses/SingleWorkspaceUnit.java

```diff
@@ -17,21 +16,25 @@
  * debugging a particular failure.
  */
 public class SingleWorkspaceUnit extends ClearContextUnit {
-  protected static final TestUser workspaceCreator = TestUser.chooseTestUserWithSpendAccess();
   private static String userFacingId;
 
   protected static String getUserFacingId() {
     return userFacingId;
   }
 
+  private String platformStorageName;
+
   @BeforeAll
   protected void setupOnce() throws Exception {
-    TestContext.clearGlobalContextDir();
-    resetContext();
+    super.setupOnce();
+    if (getCloudPlatform() == CloudPlatform.GCP) {
+      platformStorageName = "gcs-bucket";
+    }
```

Comment:
it looks like this test will break if it's run on a non-gcp platform - can this have an `else { throw some exception }` to make the failure case clearer?

Reply:
Done, Added an else clause with exception.